cloud.foundry.cli/src/main/java/cloud/foundry/cli/crosscutting/util/YamlCreator.java

```diff
@@ -12,34 +14,29 @@
 import java.util.HashMap;
 
 public class YamlCreator {
-
     /**
-     * Factory function creating a yaml processor with common options for dumping (ensures a consistent output format).
+     * Factory function creating a yaml processor with common options for dumping
+     * (ensures a consistent output format).
+     * 
      * @return yaml processor preconfigured with proper dumping options
      */
     public static Yaml createDefaultYamlProcessor() {
         DumperOptions options = new DumperOptions();
-
```

Comment:
What kind of auto-formatter removes all the whitespace explicitly added to increase readability? Please restore those lines.